stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/images/AttachmentGallery.kt

```diff
@@ -0,0 +1,68 @@
+package io.getstream.chat.android.ui.images
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.content.res.use
+import androidx.core.view.isVisible
+import androidx.fragment.app.FragmentActivity
+import androidx.viewpager2.widget.ViewPager2
+import io.getstream.chat.android.ui.R
+import io.getstream.chat.android.ui.databinding.StreamAttachmentGalleryBinding
+
+public class AttachmentGallery : ConstraintLayout {
+
+    private val binding = StreamAttachmentGalleryBinding.inflate(LayoutInflater.from(context), this, true)
+
+    private var countText: String = "%s - %s"
+
+    public constructor(context: Context) : super(context) {
+        init(null)
```

Comment:
Why do you invoke it if it do nothing?

Reply:
Just a mistake, I took it from draft too early. Let me fix this.